packages/domain-picker/src/domain-picker/style.scss

```diff
@@ -232,8 +232,7 @@ $accent-blue: #117ac9;
 }
 
 .domain-picker__suggestion-item-name {
-	flex-grow: 1;
-	flex-basis: 2px;
+	width: max-content;
```

Comment:
Internet Explorer doesn't provide support for `max-content` (see [MDN Docs](https://developer.mozilla.org/en-US/docs/Web/CSS/max-content#browser_compatibility)). Should we consider this an issue?